src/Adapter/Country/AbstractCountryHandler.php

```diff
@@ -49,4 +64,101 @@ protected function validateCountryFields(Country $country): void
             throw new CountryConstraintException('Country contains invalid field values');
         }
     }
+
+    /**
+     * Get legacy country
+     *
+     * @param CountryId $countryId
+     *
+     * @return Country
+     *
+     * @throws CountryNotFoundException
+     */
+    protected function getCountry(CountryId $countryId): Country
```

Comment:
Is this method necessary? the CountryRepository->get() already does the same, no? :thinking: